transfer/requirements.md

```diff
@@ -31,20 +31,23 @@ If desired, additional data elements may be supplied by the donor organization.
 
 ## Notifications
 
-During the process of transferring digital records to the custody of the Rockefeller Archive Center, messages will be generated to allow humans and machines to understand the status of transfers, trigger processes, and take appropriate actions to correct any errors. Transfer notification messages indicate either success or error, and are provided in formats suitable for both machine and human consumption.
+During the process of transferring digital records to the custody of the Rockefeller Archive Center, messages will be generated to allow humans and machines to understand the status of transfers, trigger processes, and take appropriate actions to correct any errors. Transfer notification messages indicate either success or error, and are provided in formats suitable for both machine and human consumption. We have modeled the machine-actionable notifications after [W3's Activity Streams 2.0 specification](https://www.w3.org/TR/activitystreams-core/).
```

Comment:
I would suggest changing the last sentence to:
Machine-actionable notifications comply with the [W3's Activity Streams 2.0 specification](https://www.w3.org/TR/activitystreams-core/).